Dockerfile

```diff
@@ -4,8 +4,19 @@
 #################### BASE BUILD IMAGE ####################
 FROM nvidia/cuda:12.1.0-devel-ubuntu22.04 AS dev
 
+# Set the DEBIAN_FRONTEND variable to noninteractive to avoid interactive prompts
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Preconfigure tzdata for US Central Time (build running in us-central-1 but this really doesn't matter.)
+RUN echo 'tzdata tzdata/Areas select America' | debconf-set-selections \
+    && echo 'tzdata tzdata/Zones/America select Chicago' | debconf-set-selections
+
 RUN apt-get update -y \
```

Comment:
Nice, it would be great to leave a command above this block saying something along the lines
```
# We install an older version of python here for testing to make sure vllm works with older versions of Python
```

to make it clear that this is not the version of Python that will be shipped for the production image (if my understanding here is correct) :)

Reply:
Great idea!